app/Models/Category.php

```diff
@@ -22,7 +22,7 @@ class FreshRSS_Category extends Minz_Model {
 	private $nbFeeds = -1;
 	/** @var int */
 	private $nbNotRead = -1;
-	/** @var array<FreshRSS_Feed>|null */
+	/** @var array<FreshRSS_Feed> */
```

Comment:
I am not sure this is a good solution, as we need to test for nullability below: `if ($this->feeds === null)`

Reply:
got to matermost ?

Reply:
$feedDAO->listByCategory($this->id()); return always a array() of FreshRSS_Feed

Reply:
https://github.com/FreshRSS/FreshRSS/pull/5434/commits/2a0cbaaf03a4f56031ded209f8920c3322dc513c